docs/dev.md

```diff
@@ -57,6 +57,12 @@ Then you can install the dependencies:
 $ yarn
 ```
 
+Install the fixtures:
+
+```console
+$ yarn fixtures -u http://cozy.localhost:8080
```

Comment:
I don't know how ACH is handled as dep in cozy-banks, but latest version of ACH use cozy.localhost:8080 as the default url since https://github.com/cozy/ACH/commit/c7bc92cfbd5d79f946c3d579794095c2cdbcc625

Reply:
oui inutile de la rajouter ici, tu as dû le faire toi car tu n'es pas en localhost, mais ce n'est pas à mettre dans la doc, ça concerne plutôt la doc d'ACH

Reply:
@JF-Cozy : I don't understand why I faced this issue. What does "tu n'es pas en localhost" ? Do you mean I need to have ACH project running in local?

Reply:
ah pardon j'ai mal lu j'ia cru que t'avais mis `-u http://cozy.tools:8080`. Le `-u http://cozy.localhost:8080` est implicite, c'est la config par défaut d'ACH. Si ça ne marche pas, c'est qu'il y a un bug côté ACH

Reply:
I don't think of a bug in ACH. I think that the ACH version is not up to date. 

Maybe you should use `yarn ach` instead of `ACH` globally and have the right version here https://github.com/cozy/cozy-banks/blob/master/package.json#L101 (since we recommended to have ACH globally I don't know why this line is there ATM?)

Reply:
Actual cozy-ach  : 1.22.0
last ACH : 1.40.0

should we upgrade?

Reply:
I don't know if the right fix is to upgrade or not. 

From ACH : 
https://github.com/cozy/ACH#install
> Install ACH using yarn or npm.
$ yarn global add cozy-ach
$ npm install -g cozy-ach

We ask to install it globally. So... I don't know if we should have it added in devDep here or not. Or if we should have an auto-upgrade mechanism inside the ACH's binary to be sure to always have the latest version... 

I really don't know

Reply:
ACH n'est pas sur toutes les app + le fait que ce soit marqué "global" dans la doc me fait dire qu'on devrait même supprimer ACH des dep de banks